docs/documentation/concepts/index.md

```diff
@@ -6,29 +6,43 @@ layout: documentation
+The functional view includes focusing on how rules effect representations of physical devices in software.
+ Perhaps most important to you, the functional view focuses on how an action in a user interface affects the software associated with the physical device it represents.
 
-**Items** represent (fine-grained) functionality that is used by applications - as user interfaces or automation logic. Items have a state and they can receive commands.
+It is a bit of an over-simplification, but you can think of the physical view as being a view of the 'real world', and the functional view being a view of the 'software world'.
 
-The glue between Things and Items are **Links**. Links are associations between exactly one Thing Channel and one Item.
-If a Channel is linked to an Item, it is "enabled", which means that the functionality that the Item represents is handled through the given Channel.
-Channels can be linked to multiple Items and Items can be linked to multiple Channels.
+## Things, Channels, Items and Links
 
-To illustrate these concepts, take a two-channel actuator that controls two lights:
+**Things** are entities that can be physically added to a system.
+Things may provide more than one function (for example, a Z-Wave multi-sensor may provide a motion detector and also measure room temperature).
+Things do not have to be physical devices; they can also represent a web service or any other manageable source of information and functionality.
 
-![](images/thing-devices-1.png)
+Things expose their capabilities through **Channels**.
+Whether an installation takes advantage of a particular capability reflected by a Channel depends on whether it has been configured to do so.
+When you configure your system, you do not necessarily have to use every capability offered by a Thing.
+You can find out what Channels are available for a Thing by looking at the documentation of the Thing's **Binding**.
+Bindings can be thought of as software adapters, making Things available to your home automation system.
 
-The actuator is the _Thing_. This might be installed in the electrical cabinet, it has a physical address and needs to be setup and configured in order to be used.
-The user is instead interested in the two lights, which are located at different locations in his home. These lights are the desired functionality, thus the _Items_ and they are linked to the _Channels_ of the actuator.
-A _Link_ can be regarded like a physical wire in this example.
+**Items** represent capabilities that can be used by applications, either in user interfaces or in automation logic.
+Items have a **State** and they may receive commands.
 
+The glue between Things and Items are **Links**.
+A Link is an association between exactly one Channel and one Item.
+If a Channel is linked to an Item, it is "enabled", which means that the capability the Item represents is accessible through that Channel.
```

Comment:
Hm, the Channel does not access the Item. Or do he?

Reply:
When I did the edit, I tried to clarify the original wording without changing the meaning.  

What I see here is that a Thing can have multiple Capabilities that are exposed through Channels.  If you want to get at a Capability of a Thing, then you have to Link that Channel of that Thing to an Item.  Once you have done that, you can control that Capability of that Thing using the linked Item.  Isn't that correct?  Do you have a different wording suggestion?